src/store/modules/user/actions.ts

```diff
@@ -290,256 +264,6 @@ const actions: ActionTree<UserState, RootState> = {
     commit(types.USER_OMS_REDIRECTION_INFO_UPDATED, payload)
   },
 
-  // This setting is intended for temporary use to enable a more controlled rollout of the
-  // reservationFacilityId Solr field changes on a client-by-client basis.
-  // It should be removed once all clients' OMS instances have been upgraded to a version that includes this change.
-  async getReservationFacilityIdFieldConfig ({ commit }) {
-    let isEnabled = false;
-
-    const params = {
-      "productStoreId": getProductStoreId(),
-      "settingTypeEnumId": "USE_RES_FACILITY_ID",
-      "fieldsToSelect": ["productStoreId", "settingTypeEnumId", "settingValue"],
-      "pageSize": 1
-    } as any
-
-    try {
-      const resp = await UserService.getReservationFacilityIdFieldConfig(params)
-      if (!hasError(resp)) {
-        isEnabled = resp.data[0]?.settingValue === "Y" ? true : false
-      } else {
-        throw resp.data;
-      }
-    } catch (err) {
-      logger.error('Failed to fetch reservation facility id field configuration');
-    } 
-    commit(types.USER_RESERVATION_FACILITY_ID_FIELD_CONFIG_UPDATED, isEnabled);   
```

Comment:
Remove the mutation, state and getters as we have move the product store setting to util.